src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -96,49 +95,38 @@ export default class LimitedRestorationPanel extends Vue {
   }
 
   /**
-   * Called when months input has changed.
+   * Called when radio value or input value have changed.
    */
-  async onMonthsInput (input: string): Promise<void> {
-    // ignore non-inputs
-    if (input !== null) {
-      // wait for component updates
+  @Watch('radioValue')
+  @Watch('inputValue')
+  private async onValueChanged (): Promise<void> {
```

Comment:
The previous 2 methods fired alternately and not always in the same order due to waiting for the input component to update/validate.

This now handles things more cleanly.